src/app/core/services/auth.service.ts

```diff
@@ -69,7 +71,6 @@ export class AuthService {
     this.githubService.reset();
     this.userService.reset();
     this.issueService.reset(true);
-    this.labelService.reset();
```

Comment:
What's the reason for removing this label service?

Reply:
I'm not sure why, but at the time I was under the impression that it wasn't necessary, which I realize might not be the case.

I've made a commit reverting the changes, sorry about that. ">_>